gateway/assets/src/error.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <meta name="description" content="A directory of content-addressed files hosted on IPFS">
+  <meta property="og:title" content="Files on IPFS">
+  <meta property="og:description" content="{{ .Path }}">
+  <meta property="og:type" content="website">
+  <meta property="og:image" content="https://gateway.ipfs.io/ipfs/QmSDeYAe9mga6NdTozAZuyGL3Q1XjsLtvX28XFxJH8oPjq">
+  <meta name="twitter:title" content="{{ .Path }}">
+  <meta name="twitter:description" content="A directory of files hosted on the distributed, decentralized web using IPFS">
+  <meta name="twitter:card" content="summary_large_image">
+  <meta name="twitter:image" content="https://gateway.ipfs.io/ipfs/QmSDeYAe9mga6NdTozAZuyGL3Q1XjsLtvX28XFxJH8oPjq">
+  <meta name="twitter:creator" content="@ipfs">
+  <meta name="twitter:site" content="@ipfs">
+  <meta name="image" content="https://gateway.ipfs.io/ipfs/QmSDeYAe9mga6NdTozAZuyGL3Q1XjsLtvX28XFxJH8oPjq">
```

Comment:
Directory description and other social media items should be removed from error page, or customized to reflect something went wrong, so if someone pastes URL with truncated CID on twitter, they get useful error in  preview.

Reply:
I removed the social media from all the three pages for the following reasons:

1. The linked image gives `410 Gone` so it's not accessible.
2. Social media will fallback on default `<meta name=description` and `<title` tags, which are kept.
3. We probably don't want all the content shared by random people to be associated with `@ipfs` on Twitter. Who knows what people are sharing around

I updated the remaining pages' descriptions to better reflect what's happening.